pkg/log/options.go

```diff
@@ -24,15 +24,48 @@ import (
 const (
 	// None is used to disable logging output as well as to disable stack tracing.
 	None zapcore.Level = 100
+
+	defaultOutputLevel        = zapcore.InfoLevel
+	defaultStackTraceLevel    = None
+	defaultOutputPath         = "stdout"
+	defaultErrorOutputPath    = "stderr"
+	defaultRotationMaxAge     = 30
+	defaultRotationMaxSize    = 100 * 1024 * 1024
+	defaultRotationMaxBackups = 1000
 )
 
 // Options defines the set of options supported by Istio's component logging package.
 type Options struct {
 	// OutputPaths is a list of file system paths to write the log data to.
 	// The special values stdout and stderr can be used to output to the
-	// standard I/O streams.
+	// standard I/O streams. This defaults to stdout.
 	OutputPaths []string
 
+	// ErrorOutputPaths is a list of file system paths to write logger errors to.
+	// The special values stdout and stderr can be used to output to the
+	// standard I/O streams. This defaults to stderr.
+	ErrorOutputPaths []string
+
+	// RotateOutputPath is the path to a rotating log file. This file should
+	// be automatically rotated over time, based on the rotation parameters such
+	// as RotationMaxSize and RotationMaxAge. The default is to not rotate.
+	RotateOutputPath string
```

Comment:
I like this much better! :)

The only thing that's still a little unclear to me is how log file naming should work, based on these options (and the supporting comments). The fact that this is an `OutputPath` and not an `OutputDir` leads me to initially think that there will only be a single rotating log file generated (which I name explicitly here). The other parameters (max size, age, etc.), however, suggest that multiple files will be generated by the rotation process.  Perhaps it would make more sense to make this a directory rather than a file? Or was that your intent?